<!DOCTYPE html>
<html lang="en">
<head>
	
	<title>People </title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="Center For Membrane Study,Lehigh University" content="">
  <meta name="Rafsan Mobasher" content="">
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="css/animate.min.css" rel="stylesheet"> 
  <link href="css/animate.css" rel="stylesheet"> 
  <link href="css/font-awesome.min.css" rel="stylesheet">
  <link href="css/lightbox.css" rel="stylesheet">
  <link href="css/main.css" rel="stylesheet">
  <link id="css-preset" href="css/presets/preset1.css" rel="stylesheet">
  <link href="css/responsive.css" rel="stylesheet">


  
  <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet' type='text/css'>
  <link rel="shortcut icon" href="images/favicon.ico">

  <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet' type='text/css'>
  <link rel="shortcut icon" href="images/favicon.ico">

  <link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="images/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="images/favicon-16x16.png" sizes="16x16">
  <link rel="manifest" href="images/manifest.json">
  <link rel="mask-icon" href="images/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="images/favicon.ico">
  <meta name="msapplication-config" content="images/browserconfig.xml">
  <meta name="theme-color" content="#ffffff">

<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
  <style type="text/css">



.navbar-brand {
  margin-bottom: 5%;
}

    #main-container{
      margin: 5% auto;
      width: 90%;
    }



    #main-container p { 
      font-family: 'Dosis', sans-serif;
      line-height: 1.88889;
      color: #555753;  
      text-align: justify;
      font-height: 20px;
      font-size:15px;
    }

    h3 { 
      font-size:3em;
      font-family: 'Lobster', cursive;
      letter-spacing: 1px; 
      margin-bottom: 0; 
      color: #7D775C;
    }
    a:link { 
      font-weight: bold; 
      text-decoration: none; 
      color: #B7A5DF;
    }
    a:visited { 
      font-weight: bold; 
      text-decoration: none; 
      color: #D4CDDC;
    }
    a:hover, a:focus, a:active { 
      text-decoration: underline; 
      color: #9685BA;
    }


    table { 
      border-spacing: 20px;
      border-collapse: separate;
      font: 75% georgia, sans-serif;
    }



    td img{
      width:100px;
    }


   /*   display: block;*/
#bs-example-navbar-collapse-1 ul a {
   color: #fff;
   font-weight: 600;
   font-family: 'Open Sans', sans-serif;
    

 }
  </style>


</head>


<body>

  <nav class="navbar navbar-default navbar-fixed-top" style="background-color: #4b2c11">
    <div class= "container-fluid">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header">
        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
      <!--   <a class="navbar-brand" href="http://www1.lehigh.edu/"><img src="images/lehigh1.png"></a> -->
        <a class="navbar-brand" href="index.html"><img src="images/logo.png"></a>
      </div>

      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">

        <ul class="nav navbar-nav navbar-right" id="nav-manu-right">
          <li class="nav-link"><a href="index.html">HOME</a></li>
          <li class="nav-link"><a href="research.html">RESEARCH</a></li>
          <li class="nav-link"><a href="people.html">PEOPLE</a></li>
          <li class="nav-link"><a href="publications.html">PUBLICATIONS</a></li>
          <li class="nav-link"><a href="giving.html">GIVING</a></li>
          <li class="nav-link"><a href="contact.html">CONTACT</a></li>
        </ul>

      </div><!-- /.navbar-collapse -->
    </div><!-- /.container-fluid -->
  </nav>

  <div id="main-container">
    <table >
      <tr>
        <td> <img src="images/team/Berger.jpg" ></td>
        <td>  <h3 class="post-title">Bryan Berger </h3>
          <br>
          <p>The Berger lab seeks to understand the structural basis by which receptors transduce signals across cell membranes. To achieve this understanding, we develop new experimental tools to interrogate receptor structure and dimerization in cell membranes, and use these tools to develop structural models for how receptors interact within a cell membrane. We also use this knowledge to design peptide-based probes that activate or inhibit receptor signaling. </p>

          <h5><a href="http://www.lehigh.edu/berger">Website</a> &nbsp;|&nbsp;<a href="mailto:bwb209@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=A37Uj4kAAAAJ&hl=en&oi=ao">Google Scholar</a></h5>
        </td>
      </tr>

      <tr>
        <td><img src="images/team/angela_brown.jpg" /> </td>
        <td> 
          <h3 class="post-title">Angel Brown</h3>
          <br>
          <p style="display: inline-block;">The Brown lab works to develop strategies to inhibit bacterial pathogenesis, with a focus on inhibition of bacterial toxin activity.  We investigate the mechanisms by which bacterial toxins recognize and interact with target cell membranes to identify therapeutic targets. </p>
          <h5><a href="http://www.lehigh.edu/engineering/faculty/profiles/cheme/brown.html">Website</a> &nbsp;|&nbsp;<a href="mailto:acb313@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=94TG3N0AAAAJ&hl=en">Google Scholar</a></h5>
        </td>    
      </tr> 
      <tr>
        <td><img src="images/team/javier.jpg"  /></td>
        <td> <h3 class="post-title">Javier Buceta </h3>
          <br>
          <p>The Buceta lab is focused on Systems Biology approaches towards the understanding of problems at different spatiotemporal scales. Our interests include the understanding of mechanosensitive properties during cell division regulated by membrane channels and the characterization of membrane proteins involved in phage infection. Our methodology combines mathematical modeling, numerical simulations, microfluidics, and microscopy at the submicron scale.</p>
          <h5><a href="http://www.thesimbiosys.com">Website</a> &nbsp;|&nbsp;<a href="mailto:jbuceta@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=IE64vjgAAAAJ&hl=en">Google Scholar</a></h5>
        </td>    
      </tr>



      <tr>

        <td><img src="images/team/xuan_cheng.jpg"  />
          <div class="post-content"></td>
            <td><h3 class="post-title">Xuanhong Cheng</h3>

             <br>

             <p>The Cheng lab is focused on developing new nano-materials and microfluidic platforms that can be used to analyze intact bioparticles, such as cells and pathogens at the point of need. Membrane properties of these bioparticles play critical roles in developing separation and detection strategies. These technologies are also expected to advance fundamental understanding of cell function and pathogen transmission.</p>

             <h5><a href="http://cheng.web.lehigh.edu">Website</a> &nbsp;|&nbsp;<a href="mailto:xuc207@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=HFqFnOsAAAAJ&hl=en">Google Scholar</a></h5>
           </h5>
         </td>
       </tr>


       <tr>
        <td><img src="images/team/LesleyChow.jpg" /></td>
        <td>
          <h3 class="post-title">Lesley Chow </h3>

          <br>

          <p>The Chow lab focuses on the synthesis and fabrication of modular biomaterials for tissue engineering and regenerative medicine. We are particularly interested in developing glycosaminoglycan-based hydrogels that mimic the glycocalyx and extracellular matrix (ECM) microenvironment to influence cell behavior. These biomaterials will expand our understanding of membrane-ECM interactions and potentially lead to new biomedical therapies. </p>

          <h5><a href="https://thechowlab.com">Website</a> &nbsp;|&nbsp;<a href="mailto:lac415@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=-4BRKdsAAAAJ&hl=en">Google Scholar</a></h5>
        </h5>
      </td>
    </tr>


    <tr>

      <td><img src="images/team/Falk.jpg"  /></td>
      <td>
        <h3 class="post-title">Matthias Falk  </h3>

        <br>
        
        <p>The Falk lab investigates the biosynthesis and function of the gap junction forming proteins, connexins, and how cells internalize and regulate their turnover to balance the level of direct cell-cell communication maintained by gap junction channels. In principle, all cells in our tissues and organs have gap junctions and their crucial role for multi-cellular function is demonstrated by multiple diseases that have been linked to mutations in connexins. Their ability to dock head-on in the extracellular space and to cluster into arrays (gap junction plaques) makes connexins particularly interesting membrane proteins.
        </p>

        <h5><a href="http://www.lehigh.edu/~inbioe/faculty/falk.html">Website</a> &nbsp;|&nbsp;<a href="mailto:mmf4@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=ummiFU8AAAAJ&hl=en">Google Scholar</a></h5>
        
      </td>




    </tr>


    <tr>

      <td><img src="images/team/glover.jpg"  /></td>

      <td>  <h3 class="post-title">Kerney Jebrell Glover</h3>

       <br>

       <p>The primary concern of the Golver Lab research is the investigation and characterization of membrane proteins and their interactions with the lipid bilayer. Of particular interest to our research group is caveolin­1, a membrane protein that is pivotal in the formation of plasma membrane invaginations known as caveolae. </p>

       <h5><a href="https://gloverlab.cas2.lehigh.edu/">Website</a> &nbsp;|&nbsp;<a href="mailto:kjg206@lehigh.edu
        ">Email</a> &nbsp;</h5>
      </td>



    </tr>



    <tr>
      <td>

        <img src="images/team/honerkamp.jpg"  /> </td>

        <td> <h3 class="post-title">Aurelia Honerkamp-Smith  </h3>
         <br>


         <p>The Honerkamp-Smith lab uses microfluidics and fluorescence microscopy to experimentally investigate how lipid membranes respond to fluid flow. We explore how flow can sort and align membrane proteins and reveal membrane physical properties.</p>

         <h5><a href="https://sites.google.com/a/lehigh.edu/honerkamp-smith/honerkamp-smith-homepage">Website</a> &nbsp;|&nbsp;<a href="mailto:auh216@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=CdeyDuYAAAAJ&hl=en">Google Scholar</a></h5>
       </h5>
     </td>

   </tr>
   <tr>

     <td><img src="images/team/wonpil_im.jpg" /></td>
     <td>
      <h3 class="post-title">Wonpil Im </h3>

      <br>

      <p>Research in Im lab is focused on the applications of theoretical/computational methods to chemical and physical problems in biology and material sciences. In particular, Im lab is interested in modeling and simulations of biological membranes and associated proteins, glycoconjugates, and protein-ligand (drug) interactions. In addition, Im lab has been developing <a href="http://www.charmm-gui.org">CHARMM-GUI</a> for the biomolecular modeling and simulation community.</p>

      <h5><a href="http://compbio.lehigh.edu">Website</a> &nbsp;|&nbsp;<a href="mailto:wonpil@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=qp8birIAAAAJ">Google Scholar</a></h5>
    </td>
  </tr>



  <tr>
    <td><img src="images/team/1.jpg"  /></td>
    <td>
      <h3 class="post-title">M. Kathryn Iovine </h3>

      <br>

      <p>Research in the Iovine lab is focused on skeletogenesis in the regenerating zebrafish fin. We have discovered that direct cell-cell communication through Cx43-based gap junctions initiates a gene regulatory network responsible for coordinating skeletal growth (i.e. cell proliferation) and skeletal patterning (i.e. joint formation). Changes in the composition and structure of the extracellular matrix occur downstream of Cx43, and may be responsible for connecting the distinct populations of dividing cells responsible for outgrowth and the differentiating cells responsible for skeletal patterning.
      </p>

      <h5><a href="http://www.lehigh.edu/~inbios/Faculty/Iovine.html">Website</a> &nbsp;|&nbsp;<a href="mailto:mki3@lehigh.edu">Email</a> &nbsp;</h5>
    </td>
  </tr>

  <tr>

    <td><img src="images/team/anand.jpg"  />
      <div class="post-content"></td>
        <td>
          <h3 class="post-title">Anand Jagota</h3>

          <br>

          <p>The organizing theme for the Jagota Research Group is the study of problems in the processing and properties of materials, especially those related to bioengineering. We combine theory and experiments in our work. Our current interests include hybrids of biological molecules and nanomaterials, biomimetic design of surface mechanical properties, and the mechanics of soft materials.
          </p>

          <h5><a href="http://www.lehigh.edu/~anj6/">Website</a> &nbsp;|&nbsp;<a href="mailto:anj6@lehigh.edu">Email</a> &nbsp;|&nbsp;<a href="https://scholar.google.com/citations?user=EoRGtkYAAAAJ&hl=en">Google Scholar</a> </h5>
        </td>   
      </tr>
      <tr>
        <td><img src="images/team/sabrina.jpg"  /></td>
        <td>
          <h3 class="post-title">Sabrina Jedlicka </h3>

          <br>

          <p>Professor Jedlicka’s research revolves around biologically functional material design for use in cell based therapeutics and drug delivery.  Biologically functional materials provide the ability to manipulate the direct extracellular environment surrounding cells; influencing their fate and differentiation path.  In addition, engineering materials that can be directed using native biological signaling pathways provides a means to direct drug delivery into the central and peripheral nervous system. 
          </p>

          <h5><a href="http://www.lehigh.edu/engineering/faculty/profiles/matsci/jedlicka.html">Website</a> &nbsp;|&nbsp;<a href="mailto:ssj207@lehigh.edu">Email</a> &nbsp;| &nbsp; <a href="https://scholar.google.com/citations?user=RiKJW3cAAAAJ&hl=en">Google Scholar</a></h5>

        </td>
      </tr>


      <tr>
        <td><img src="images/team/linda.jpg"  /></td>

        <td> <h3 class="post-title">Linda Lowe-Krentz  </h3>

         <br>

         <p>Research in the Lowe-Krentz lab aims to understand control of wound repair in vascular cells.  We are particularly interested in signal transduction pathways that involve heparan sulfate proteoglycans, the heparin receptor we recently identified, and its relationship to fluid shear stress signaling.  

         </p>

         <h5><a href="hhttp://www.lehigh.edu/~inbios/Faculty/LoweKrentz.html">Website</a> &nbsp;|&nbsp;<a href="mailto:ljl0@lehigh.edu">Email</a> &nbsp;</h5>
       </td> 

     </tr>

     <tr>

      <td><img src="images/team/mcl.jpg"  /></td>
      <td>
        <h3 class="post-title">Krystle McLaughlin </h3>

        <br>
        
        <p>Research in the McLaughlin Lab makes primary use of macromolecular x-ray crystallography, along with several biochemical and biophysical methods, to investigate protein structure and function. Research interests involve the study and characterization of protein targets from (1) the commensal human gut bacteria Bacteroides ovatus, and (2) the conjugative plasmids Staphyloccocus aureus pSK41 and Salmonella typhimurium pCU1, (3) the Mycobacterium bacteriophage Butters, and (4) the eukaryotic lynx gene. 

        </p>

        <h5><a href="http://www.lehigh.edu/bio/Faculty/McLaughlin.html">Website</a> &nbsp;|&nbsp;<a href="mailto:krm514@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=NUjxSt0AAAAJ&hl=en">Google Scholar</a></h5>
      </td>
    </tr>


    <tr>

      <td><img src="images/team/jeetain.jpg" /></td>

      <td><h3 class="post-title">Jeetain Mittal</h3>
        <br>

        
        <p> 
          Research in Mittal group is centered around broad themes of Biophysics and Soft Matter with a special emphasis on interfacial behavior. In biophysics research, our long-term objective is to develop predictive theoretical tools for the impact of dynamic cellular environment on biochemical processes such as protein folding, protein binding, and protein-DNA interactions. Elucidating the influence of cellular membranes on these processes is an integral part of our studies. 

        </p>

        <h5><a href="http://www.lehigh.edu/~jem309">Website</a> &nbsp;|&nbsp;<a href="mailto:jeetain@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=VA-cmnQAAAAJ&hl=en&oi=ao">Google Scholar</a></h5>
      </td>
    </tr>




    <tr>

      <td>    <img src="images/team/miwa.jpg"  </td>

      <td> <h3 class="post-title">Julie Miwa </h3>

       <br>

       <p>The Miwa lab investigates mechanisms of receptor regulation through a set of protein modulators in the brain. These endogenous protein modulators influence receptor assembly and composition, as well as gating properties (e.g. ligand-sensitivity, desensitization kinetics and mean open times, etc.) of ligand-gated ion channels. The influence of these modulators extend to synaptic signaling, plasticity, learning, and behavioral adaptation.
       </p>

       <h5><a href="https://www.lehigh.edu/bio/Faculty/Miwa.html">Website</a> &nbsp;|&nbsp;<a href="mailto:jmm312@lehigh.edu">Email</a> &nbsp; </h5>
     </td>
   </tr>

   <tr>
    <td><img src="images/team/piers.jpg"  /></td>
    <td>
      <h3 class="post-title">Marcos Pires </h3>
      
      <br>

      <p>The Pires laboratory have been carving out a novel path that unites chemical synthesis with microbiology and this powerful combination is opening the door for the chemical interrogation of live bacterial cells. Currently, Dr. Marcos Pires is investigating the remodeling of bacterial cell walls, a process of critical importance both in homeostasis and in pathology, using synthetic cell wall analogs by developing a clever strategy aimed at hijacking the cell wall biosynthetic machinery to install specific probes that can report on modifications by cell wall-linked enzymes. By shedding light on the processes that drive cell wall remodeling, the Pires laboratory will establish how bacterial cells respond to environmental cues. 

      </p>

      <h5><a href="http://www.lehigh.edu/~map311">Website</a> &nbsp;|&nbsp;<a href="mailto:map311@lehigh.edu">Email</a> &nbsp;</h5>
    </td>
  </tr>



  <tr>
    <td><img src="images/team/regan.jpg"  /></td>
    <td>    
      <h3 class="post-title">Steve Regen </h3>
      

      <br> 
      <p>Research in the lipids and membrane area in the Regen laboratory focus on (i) fundamental studies aimed at understanding the lateral organization of cell membranes using model systems and (ii) the design and synthesis of antibiotics that operate via membrane disruption. 

      </p>

      <h5><a href="http://www.lehigh.edu/~inslr">Website</a> &nbsp;|&nbsp;<a href="mailto:slr0@lehigh.edu">Email</a> &nbsp; </h5>
    </td>

  </tr>



  <tr>

    <td>    <img src="images/team/damien.png" /></td>
    
    <td>    <h3 class="post-title">Damien Thévenin </h3>

     <br>

     <p> Research in the Thévenin lab is organized around: (1) developing novel strategies to specifically target and deliver small therapeutic molecules and peptides to cancer cells and (2) identifying the structural requirements for the oligomerization of receptor protein-tyrosine phosphatases and their effects on cancer cell signaling. As such, our work is applied, fundamental, highly interdisciplinary, and draws upon areas such as peptide and protein chemistry, biophysics, molecular and structural biology, and cancer cell biology.


     </p>

     <h5><a href="http://www.lehigh.edu/~dat311/index.html">Website</a> &nbsp;|&nbsp;<a href="dat311@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=f502xW4AAAAJ&hl=en&oi=ao">Google Scholar</a></h5>
   </td>
 </tr>


 <tr>     
  <td><img src="images/team/dimitrios.jpg" /></td>
  <td>
    <h3 class="post-title">Dimitrios Vavylonis </h3>

    <br>

    <p> The Vavylonis group develops coarse-grained mathematical and computational models in the area of cell biophysics. Recent work with experimental collaborators includes models of local accumulation of proteins on the cell membrane for polarized cell wall growth and cell mating, the interaction of actin cytoskeletal networks with the plasma membrane and the ER during cytokinesis, and actin-polymerization-driven extension of the membrane at the leading edge of motile cells.



    </p>

    <h5><a href="http://athena.physics.lehigh.edu">Website</a> &nbsp;|&nbsp;<a href="vavylonis@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=HD1NqKwAAAAJ&hl=en">Google Scholar</a></h5>
  </td>
</tr>



<tr>
  <td><img src="images/team/dmitri.jpg" /></td>

  <td>
    <h3 class="post-title">Dmitri Vezenov </h3>
    <br>

    <p>Research in the Vezenov lab focuses on development of new tools for single molecule and single cell manipulation and analysis. This work is pursued along several objectives: (1) to develop next-generation sequencing technology based on massively parallel single molecule force spectroscopy; (2) to understand and exploit intermolecular interactions between biomolecules and nanomaterials; (3) to understand the role of mechanical properties of cells in their development and viability.


    </p>

    <h5><a href="http://www.dvezenovgroup.org">Website</a> &nbsp;|&nbsp;<a href="dvezenov@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=gMesDq8AAAAJ&hl=en">Google Scholar</a></h5>
  </td>
</tr>



<tr>

 <td><img src="images/team/vassie.jpg"  /></td>
 <td>
  <h3 class="post-title">Vassie Ware </h3>

  <br> 

  <p> The Ware lab uses biochemical, cellular, genetic, and molecular approaches to pursue two major research goals: (1) to understand the differential expression and function of Drosophila melanogaster eRpL22 ribosomal protein paralogues to determine if paralogues specify heterogeneous populations of ribosomes with specialized functions in translation and/or if novel or extraribosomal protein functions can be identified; (2) to characterize the role of novel mycobacteriophage-encoded membrane proteins in bacterial defense mechanisms against viral attack.  


  </p>

  <h5><a href="https://www.lehigh.edu/bio/Faculty/Ware.html">Website</a> &nbsp;|&nbsp;<a href="vcw0@lehigh.edu">Email</a> &nbsp; </h5>
</td>
</tr>


<tr>
  <td> <img src="images/team/nathan.jpg" /></td>
  <td>
    <h3 class="post-title">Nathan Wittenberg</h3>

    <br>

    <p> The Wittenberg Lab focuses on analytical chemistry applied to biological systems. We are developing new analytical methods for monitoring biomolecular interactions on surfaces, with a number of applications in neuroscience. We are also building microfluidic and nanofabricated platforms for detecting molecular transport in membrane-bound biological nanostructures like extracellular vesicles, secretory vesicles, and synaptosomes. 



    </p>

    </h5 ><a href="http://bioanalysis.cas2.lehigh.edu/" style="font-size: 1.3em">Website</a> &nbsp;|&nbsp;<a href="njw@lehigh.edu" style="font-size: 1.3em">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=DDJ3cdIAAAAJ&hl=en" style="font-size: 1.3em">Google Scholar</a></h5>
</td>
</tr>



<tr>

  <td><img src="images/team/frank.jpg"  /></td>
  <td>
    <h3 class="post-title">Frank Zhang </h3>

    <br> 

    <p> Xiaohui "Frank" Zhang leads leads an interdisciplinary Lehigh research team that integrates physics, immunology and biology toward the development of a nanodevice that could provide a new treatment to a host of human disease. His team specializes in mechanosensing -- how cells sense and respond to mechanical stimuli. Their efforts focus on the biomechanical mechanisms of protein folding, conformational change and protein-protein interactions. Zhang’s group is also active in the development and application of single-molecule force spectroscopic approaches to study cell adhesion and migration.


    </p>

    <h5><a href="http://www.lehigh.edu/~inmem/faculty/zhang.html">website</a> &nbsp;|&nbsp;<a href="xiz310@lehigh.edu">Email</a> &nbsp; | &nbsp; <a href="https://scholar.google.com/citations?user=eQtvyCoAAAAJ&hl=en&oi=ao">Google Scholar</a></h5>
  </td>

</tr>






</table>



</div><!--main-container-->




<footer id="footer" style="background-color: #4b2c11">
  <div class="footer-top wow fadeInUp" data-wow-duration="1000ms" data-wow-delay="300ms">
    <div class="container text-center">
      <div class="footer-logo">
        <a href="index.html"><img class="img-responsive" src="images/logo2.png" alt=""></a>
      </div>

    </div>
  </div>
  <div class="footer-bottom">
    <div class="container">
      <div class="row">
        <div class="col-sm-6">
          <p style="color: #4b2c11">&copy; 2017 CMS, Lehigh University.</p>
        </div>
        <div class="col-sm-6">
          <p class="pull-right" style="color: #4b2c11">Created by <a href="http://www.rafsanmobasher.com/" style="color: #4b2c11; font-weight: 100">Rafsan Mobasher</a></p>
        </div>
      </div>
    </div>
  </div>
</footer>

<script type="text/javascript" src="js/jquery.js"></script>
<script type="text/javascript" src="js/bootstrap.min.js"></script>
<script type="text/javascript" src="http://maps.google.com/maps/api/js?sensor=true"></script>
<script type="text/javascript" src="js/jquery.inview.min.js"></script>
<script type="text/javascript" src="js/wow.min.js"></script>
<script type="text/javascript" src="js/mousescroll.js"></script>
<script type="text/javascript" src="js/smoothscroll.js"></script>
<script type="text/javascript" src="js/jquery.countTo.js"></script>
<script type="text/javascript" src="js/lightbox.min.js"></script>
<script type="text/javascript" src="js/main.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>



</body>
</html>